uint8[] ax_id
uint8[] xl_id

float32[] ax_position
float32[] xl_position